source/_data/projects.yml

```diff
@@ -0,0 +1,33 @@
+# open source projects that use Cypress to write end-to-end tests
+
+- name: Redash
+  description: "Connect and query your data sources, build dashboards to visualize data and share them with your company."
+  url: https://github.com/getredash/redash/tree/master/client
+
+- name: Mermaid.js
+  description: "Generation of diagram and flowchart from text in a similar manner as markdown"
+  url: https://github.com/mermaid-js/mermaid
+
+- name: WithSpectrum
+  description: "Simple, powerful online communities at spectrum.chat"
+  url: https://github.com/withspectrum/spectrum
+
+- name: Vue Storefront
+  description: "PWA for eCommerce that is 100% offline, platform agnostic, and headless"
+  url: https://github.com/DivanteLtd/vue-storefront/tree/master/test/e2e
+
+- name: Netlify CMS
+  description: "A CMS for Static Site Generators at netlifycms.org"
+  url: https://github.com/netlify/netlify-cms/tree/master/cypress
+
+- name: ngx-bootstrap
+  description: Fast and reliable Bootstrap widgets in Angular
+  url: https://github.com/valor-software/ngx-bootstrap/tree/development/cypress
+
+- name: "octokit/rest.js"
+  description: GitHub REST API client for JavaScript
+  url: https://github.com/octokit/rest.js/tree/master/test
+
+- name: For All Beautiful Earth
```

Comment:
This is also putting the organization/user name instead of the project name. These should be consistent and respectful of the project's intended brand - how they advertise themselves.

Reply:
this one is a tough one - the website / org name is too long and harder to read. Official website is fab.earth which redirects to https://forallabeautifulearth.org/ - so in this case I think it makese sense to spell out the project's name